"make fmt" fails on Mac OS after making one of the "docker-*" targets

**Describe the bug**
Running `make fmt` fails on Mac OS after running `make docker-test-unit` because too many Go source files are found when evaluating `GO_FILES`:
```
GO_FILES        := $(shell find . -name '*.go')
```
This is because `make docker-test-unit` creates a `.cache` folder where all the Go dependencies are pulled. `find` descends in this directory and too many files are found, producing a shell argument list which exceeds the system limits.

**To Reproduce**
Run `make docker-test-unit` followed by `make fmt` on Mac OS.

**Expected**
`make fmt` should invoke `gofmt` on the Go source files.

**Actual behavior**
```
===> Formatting Go files <===
make: /bin/sh: Argument list too long
make: *** [fmt] Error 1
```

**Versions:**
N/A

**Additional context**
A simple fix would be to exclude the `.cache` folder from the Go source file search.

--- a/Makefile
+++ b/Makefile
@@ -3,7 +3,7 @@ GO              ?= go
 LDFLAGS         :=
 GOFLAGS         :=
 BINDIR          := $(CURDIR)/bin
-GO_FILES        := $(shell find . -name '*.go')
+GO_FILES        := $(shell find . -type d -name '.cache' -prune -o -type f -name '*.go' -print)
 GOPATH          ?= $$(go env GOPATH)
 DOCKER_CACHE    := $(CURDIR)/.cache
 

